### PR TITLE
Add an example for unbound + redis

### PR DESCRIPTION
**Pull request by IngmarStein**
Adds a Docker Compose project to illustrate how to use unbound with redis.

### CODE REVIEW

File: examples/redis/custom.conf.d/cachedb.conf [hunk 1]
```diff
@@ -0,0 +1,8 @@
+server:
+  module-config: "validator cachedb iterator"
+
+cachedb:
+  backend: "redis"
+  redis-server-host: redis
+  redis-server-port: 6379
+  redis-expire-records: yes
```
Comment (klutchell) [inline examples/redis/custom.conf.d/cachedb.conf:8]: This file should have the `.example` extension otherwise it will be loaded by default.

Comment (IngmarStein) [reply examples/redis/custom.conf.d/cachedb.conf:8]: But this is in the examples directory, not in roofs_overlay - it should not be part of for the image, right?

Comment (klutchell) [reply examples/redis/custom.conf.d/cachedb.conf:8]: Of course, my mistake!

**Comment by klutchell** [comment]
Thanks for the example!

**Pull request merged**